src/install_deps_command.sh

```diff
@@ -278,7 +286,7 @@ install_deps_macos() {
   # shellcheck disable=SC2086
   brew install ${BASIC_PACKAGES_TO_INSTALL[*]} || true
```

Comment:
Are you sure you want this `|| true`? rather than exit and tell users an error occurs.